### DISCUSSION TITLE
[Feature] Per-device default album

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#7037] [Feature Request]
### The feature

I run into issues where I either have my wife's and son's phones use their own accounts (but then if they aren't on top of curating then things get lost/bloated plus ultimately I want out photos in a single repository) or else we all use the same account but all their crap gets littered amongst my carefully curated library :)

The ideal solution, as I see it, is to have each device have an _option_ to auto-upload to a named album.

Then I can go in at my leisure and clean up those albums.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Closely related: #1678 and #7029
